core/tests/webdriverio_desktop/extensions.js

```diff
@@ -54,7 +54,7 @@ describe('rich-text components', function() {
       // TODO(Jacob): Add test for image RTE component.
       await richTextEditor.addRteComponent('Link', 'https://google.com/', true);
       await richTextEditor.addRteComponent(
-        'Video', 'M7lc1UVf-VE', 10, 100, false);
+        'Video', 'M7lc1UVf-VE', 10, 100, true);
```

Comment:
Will put up a PR soon to fix this boolean checkbox -- if `true` is causing test errors, you will be able to change it back to `false` once my PR is merged. Otherwise can leave it as `true`

Reply:
Sure, thanks

Reply:
cc @U8NWXD

Reply:
PR is here: https://github.com/oppia/oppia/pull/16271